readme.md

```diff
@@ -70,15 +74,15 @@ Once installation is working, try creating and requesting resources. Note that t
 * **Question:** Describe what the above code is adding to your schema.
 
 * **Your Answer:**
-
+It is adding the primary key, the unique value.
```

Comment:
This is not quite right. If you include the above in your code, it will require you to send an ObjectID when creating a new record.